chore: Add warning of enforcing region param (#32)
wherobots/wherobots-python-dbapi: README.md

The only supported Wherobots compute region for now is `aws-us-west-2`,
in AWS's Oregon (`us-west-2`) region.

> [!WARNING]
> To prepare for the expansion of Wherobots Cloud to new regions and cloud providers, the `region` parameter will become mandatory in a future SDK version.
> Before this support for new regions is added, we will release an updated version of the SDK.

> > If you continue using an older SDK version, your existing codes will still work.
> > However, any new or existing SQL session you create without specifying the `region` parameter will be hosted in the `aws-us-west-2` region.

Reviewer: ```suggestion
While you can continue using an older SDK version for your development, any new or existing SQL session you execute without specifying the `region` parameter will be hosted in the `aws-us-west-2` region.
```
